README.md

```diff
@@ -44,14 +44,20 @@ cd physlr/src && make install PREFIX=/opt/physlr
 
 ## Generating Physlr Physical Map with stLFR reads
 
-To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
+To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome (optional).
```

Comment:
Is the "optional" needed?

Reply:
@jowong4 I wanted to somehow remind the user that this is done _de novo_ and the reference is optional for those purposes mentioned. We can alternatively rephrase and add _de novo_ somewhere and remove this.

Reply:
On that train of thought - I wonder if we are best to only have the first sentence here, and an example command _without_ a reference genome. Then, after that, we could say that you can visualize stats with a reference, and show an example of running in that way? 
Just thinking that we want to make it clear how to run physlr without a reference, which may be the more common use-case. Don't want someone thinking that it's reference-based.

Reply:
Yeah, I was just editing and I was thinking the same way you just said... I'll switch to that one in a moment

Reply:
@lcoombe done, you can check the changes now, thank you